data/json/items/comestibles/raw_veggy.json

```
@@ -181,6 +181,26 @@
     "smoking_result": "dry_veggy",
     "vitamins": [ [ "vitC", 2 ], [ "calcium", 4 ], [ "iron", 2 ], [ "veggy_allergen", 1 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "radish_taproot",
+    "name": { "str_sp": "radish taproot" },
```

```suggestion
    "name": { "str": "radish taproot" },
```

This should have proper pluralization